docs/DeveloperGuide.md

```diff
@@ -740,58 +739,109 @@ testers are expected to do more *exploratory* testing.
 
+1. Editing a person
+    1. Prerequisites: At least one person is showing in the list.
+    2. Test case:`edit person 1 p/90909090`
+    3. Expected: The contact is edited.
+    - The status bar shows the following: "Edited person.... Phone: 90909090...."
+    - The list of persons should now have the edited contact inside it.
+
 ### Finding a person
 
+1. Finding a person
+    1. Prerequisites: At least one person in DocTrack. 
+        - For a simple example, use the person added in the test case for adding a person.
+    2. Test case:`find person n/Elmo`
+    3. Expected: The list is updated to show all the people with names containing "elmo", non-case-sensitive.
+    - The status bar shows the following: "Found _ persons".
+    - The list of persons should now only show people matching the criteria.
+
 ### Adding an appointment
 
+1. Adding a person
+    1. Prerequisites: There must be a person with the PID 1.
+    2. Test case:`add appt ty/Check up d/2024-10-16 12:30:30 i/1 s/Common Cold m/Paracetamol`
+    3. Expected: A new appointment is added to the list.
+    - The status bar shows the following: "New appointment added: Check up; Date and Time2024-10-16T12:30:30; 
+   Sickness: Common Cold; Medicine: Paracetamol
+    - The list of appointments should now have the new appointment inside it.
+
 ### Deleting an appointment
 
+1. Deleting a person while all persons are being shown
+    1. Prerequisites: Multiple appointments in DocTrack. If they are not shown, do `list appt`.
+    2. Test case: `delete appt 1`<br>
+       Expected: First contact is deleted from the list. Details of the deleted appointment shown in the status message.
+       Timestamp in the status bar is updated.
+    3. Test case: `delete appt -1 `<br>
+       Expected: No appointment is deleted. Error details shown in the status message. Status bar remains the same.
+    4. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+       Expected: Similar to previous.
+   
 ### Editing an appointment
 
+1. Editing a person
+    1. Prerequisites: At least one person is showing in the list.
+    2. Test case:`edit appt 1 ty/Health Checkup`
+    3. Expected: The appointment is edited.
+    - The status bar shows the following: "Edited Appointment: Health Checkup; ....."
+    - The list of appointments should now have the edited appointment inside it.
+
 ### Finding an appointment
 
+1. Finding an appointment
+    1. Prerequisites: At least one appointment in the DocTrack book.
+        - For a simple example, use the appointment added in the test case for adding a person.
+    2. Test case:`find appt n/[NAME]`
+    3. Expected: The list is updated to show all the appointments with names containing "[NAME]", non-case-sensitive.
+    - The status bar shows the following: "Found _ appointments".
+    - The list of appointments should now only show appointments matching the criteria.
+   
 ### Saving data
```

Comment:
Thanks for doing the test cases for manual testing! Should we include the testing for the `clear` commands as well?

Reply:
Good idea, I'll get to it.